src/handlers/motions/motionStaked/motionStaked.ts

```diff
@@ -24,7 +24,9 @@ export default async (event: ContractEvent): Promise<void> => {
   const requiredStake = getRequiredStake(skillRep, totalStakeFraction);
   const motionStakes = getMotionStakes(requiredStake, stakes);
   const remainingStakes = getRemainingStakes(requiredStake, stakes);
-
+  const showInActionsList =
```

Comment:
We put a motion in the actions list if it has been staked above 10%

Reply:
So the way I see it, they need to be in the database iregardless of the percent of stake they have, and have them filtered by that percent, only when querying via graphQL

This is because of couple of reasons:
- we might change that percent value (which means we'll not be able to show ongoing / historic motions since as far as our db is concerned they won't exist)
- there was talk at some point that we might wave that limitation for certain users (more reputation, or some permissions)

Reply:
It's not ideal that block-ingestor knows of that, but it's also hard to think about a good alternative. I feel like it'd be better to filter for this condition in the CDapp (we do it regardless for the `showInActionsList` field, and it's a UI detail that I think belongs in the data presentation) but it's not a single field that you can filter on in your query...

Maybe if we can stop writing motion-created actions in the DB (see my later question) this will be irrelevant?